.github/workflows/ci.yaml

```diff
@@ -61,6 +61,11 @@ jobs:
 
       - run: cabal test
 
+      - if: matrix.os == 'ubuntu-20.04'
+        run: npm install ajv-cli
+      - if: matrix.os == 'ubuntu-20.04'
+        run: for f in output/????.json; do echo $f; npx ajv -s output/schema -d $f; done
```

Comment:
I should probably re-write this "script" in Haskell and run it on every operating system. It would be similar to `src/util/copy-executable.hs`.